django_stubs_ext/django_stubs_ext/db/router.py

```diff
@@ -0,0 +1,32 @@
+from typing import TYPE_CHECKING, Optional, Type
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from django.db.models import Model
+
+    class TypedDatabaseRouter:
+        """
+        Typed base class for Django's DATABASE_ROUTERS setting. At runtime this is just an alias to `object`.
+
+        All methods are optional.
+
+        Django documentation: https://docs.djangoproject.com/en/dev/topics/db/multi-db/#automatic-database-routing
```

Comment:
secret tip: `stable` URLs for Django docs:
```suggestion
        Django documentation: https://docs.djangoproject.com/en/stable/topics/db/multi-db/#automatic-database-routing
```